sdk/search/search-documents/samples/typescript/src/dataSourceConnections/createDataSourceConnection.ts

```diff
@@ -12,9 +12,12 @@ dotenv.config();
 const endpoint = process.env.SEARCH_API_ENDPOINT || "";
 const apiKey = process.env.SEARCH_API_KEY || "";
 
-async function main(): Promise<void> {
+export async function main() {
```

Comment:
why is this export here in the TS samples? Since we call main below, it doesn't really make sense to do the export, as a module importing this one would cause the sample to run immediately

Reply:
Done(In the next commit)